client/Components/Cart.js

```diff
@@ -36,12 +37,12 @@ const Cart = ({ cart, products }) => {
                     </div>
                 </div></td>
                 <td className="col-sm-1 col-md-1" style={{textAlign: "center"}}>
-                <input type="email" className="form-control" id="exampleInputEmail1" value={lineItem.quantity} />
+                <input type="email" onChange={handleQuantityUpdate} className="form-control" id="exampleInputEmail1" value={lineItem.quantity} />
```

Comment:
Since this is the quantity input we should update the 'type' & 'id' so they correspond to quantity.